frontend/assessments/1/style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  overflow: hidden;
+  font-family: "Poppins", sans-serif;
+  font-size: 20px;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  width: 100%;
+  height: 200px;
+}
+.header-container #website-header {
+  width: 100%;
+  height: 50%;
+  background-color: #eb825a;
+  font-size: 2rem;
+  padding-left: 150px;
```

Comment:
Not a good practice
Instead you should have used:
width: 80%;
margin: auto;

It could have aligned your text in center without hardcoded spacing from left. Now it is looking very odd

Reply:
Noted, Will ensure the same next-time.